beetmoverscript/maven_utils.py

```diff
@@ -1,6 +1,6 @@
 import os
 
-_MAVEN_ZIP_NAME = 'target.maven.zip'
+_MAVEN_ZIP_NAME = '.maven.zip'
```

Comment:
Nit: Rename constant to `_MAVEN_ZIP_SUFFIX`?

Reply:
👍